src/models/MosaicSession.js

```diff
@@ -300,7 +321,10 @@ export default class MosaicSession {
     if (isPedigree) {
       return self.promiseGetPedigreeForSample(project_id, sample_id);
     } else {
-      return self.promiseGetSample(project_id, sample_id, 'proband');
+      return self.promiseGetSample(project_id, sample_id, 'proband')
+        .then((sample) => {
+          return self.promiseGetSampleAttributes(project_id, sample)
```

Comment:
The data returned from promiseGetSample no longer has the attributes, it was split off into a seperate route which is why we have to make two requests here. Also, I'm used to using `async/await` rather than `then` so hopefully this works. What I'm trying to do is make the first request, then make the second request, combine them and return them.